transform/app/tasks/transform/dump/process.py

```diff
@@ -0,0 +1,81 @@
+"""Main celery task for managing of processing a dump of data"""
+
+import logging
+from datetime import datetime
+from typing import Optional
+
+from celery import chain
+from celery.result import AsyncResult
+
+from app.api.schemas.transform.dump import DumpUpdateRequest
+from app.tasks.solr.create_collections import create_solr_collections_task
+from app.tasks.transform.dump.transform import transform_data
+from app.tasks.transform.dump.validate import validate_dump
+from app.worker import celery
+from app.services.celery.task import CeleryTaskStatus
+
+logger = logging.getLogger(__name__)
+
+
+@celery.task(name="update_peripheral_data")
+def update_peripheral_data(
+    prev_task_status: Optional[CeleryTaskStatus], req_body: dict
+) -> dict:
+    """Task to update peripheral data."""
+    logger.info("Updating peripheral data...")
+    # Invoke send task
+
+    return CeleryTaskStatus(status="toBeImplemented").dict()
+
+
+@celery.task(name="transform_dump")
+def process_dump(req_body: dict) -> AsyncResult:
+    """Main celery task for managing of processing a dump of data.
+    Workflow:
+        1) Validate dump
+        2)* Create solr collections
```

Comment:
Just a typo

Reply:
Actually, it is done on purpose because the second step is optional. Someone may send data only to S3

Reply:
I see, maybe there should be a side note at the end of the docstring? Like: `* - optional step'? If not, leave it as it is.

Reply:
Added " - optional "